src/components/Button/Button.tsx

```diff
@@ -9,11 +9,12 @@ type Props = {
   onClick: () => void;
 };
 
-export default function Button({label, color, onClick, variant}: Props) {
+export default function Button({ label, color, onClick, variant }: Props) {
   const buttonStyle = cn(
     "button",
-    {"button--ghost": variant === "ghost"},
-    {"button--blue": color === "blue"}
+    { "button--ghost": variant === "ghost" },
+    { "button--blue": color === "blue" },
+    { "button--cross": variant === "cross" }
```

Comment:
Let's group by type, like so:

```suggestion
    { "button--ghost": variant === "ghost" },
    { "button--cross": variant === "cross" },
    { "button--blue": color === "blue" }
```

This way it's better to identify all possibilities for this option